README.md

```diff
@@ -1,24 +1,52 @@
-# README
+## Forms Project
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+    This project puts our knowledge of forms, models, controllers, and using form_helpers to the tests. It is an application where a user can create a user and edit it.
 
-Things you may want to cover:
+Additional description Odin Project
+Built With
 
-* Ruby version
+    Ruby on Rails
 
-* System dependencies
+## Getting Started
 
-* Configuration
+To get the local copy of this project you just need to clone it by running the following command from your terminal :
+- git clone https://github.com/safafa/re-former/tree/form1
+- make sure you have rails already installed in your machine
 
-* Database creation
+## Installation :
 
-* Database initialization
+After you have cloned the repository, you need to :
+- Open terminal make sure you're in the directory of the project cloned
+- run : <code> bundle install </code> to install all the available gems 
+- run : <code> rails db:migrate </code> to migrate the database
+- run : <code> rails server </code> to start the server
+Setup
```

Comment:
As requested by the last reviewer. You need to write detailed and complete instructions required to run the app locally. The instructions should be clear enough even for nonprogrammers.  Let me continue from where you stop.
enter `http://localhost:3000/users/new` to create new users
enter `http://localhost:3000/users/1/edit` to edit existing users